libs/miner/src/basic_miner.cpp

```diff
@@ -151,7 +151,7 @@ void BasicMiner::GenerateBlock(Block &block, std::size_t num_lanes, std::size_t
     std::vector<QueuePtr> transaction_lists(num_threads);
```

Comment:
Since the constructor is no longer needed here we can make this a `std::vector<Queue>` and remove `num_threads` extra allocations

Reply:
Nice.